src/actions/exchangeActions.js

```diff
@@ -379,3 +385,55 @@ export const getExchangeSupportedAssetsAction = (callback?: () => void) => {
     dispatch(saveDbAction('exchangeSupportedAssets', { exchangeSupportedAssets }, true));
   };
 };
+
+export const getWbtcFeesAction = () => (dispatch: Dispatch) => {
+  fetch(getEnv().WBTC_FEES_API, {
```

Comment:
We stopped using `fetch` due timeouts not working and all the API calls are made with `axios` (except HTML content load).

Reply:
Error is not thrown on `fetch` timeout as well, it will return as successful promise, you might want to change this to `axios` for more control.